scheduler/src/cook/mesos/util.clj

```diff
@@ -26,6 +26,44 @@
             [plumbing.core :as pc :refer (map-vals map-keys)])
   (:import [java.util Date]))
 
+(defn lookup-cache-generic!
```

Comment:
Are there any uses for this besides `lookup-cache-datomic-entity!`? If so, I'd rather just have the one function (which can be private)

Reply:
Right now, no. But its not unlikely I may add a cache thats not based on lookup-cache-datomic-entity!

Reply:
Given our conversation yesterday, that sounds good to me. This could also be an interface for a guava cache as well, so it's fine with me to leave it.